topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
-> - 2b. Click on “Choose from repository” at the bottom of the popup
+> <hands-on-title>Get your data from your computer</hands-on-title>
+> - Click on **Upload** in the vertical panel on the left
+> - In the pop-up, click on **Choose local file**
+> - Select the files on your system (Argo float or WOA)
+> - Click on **Open** then **Start**
+> - Wait until all files are stored in your history (green color)
 >
-> ![Image of how to uploade remote data files](../../images/bgc_calib/bgc_2a.png)
+> ![Local upload interface](../../images/bgc_calib/local_upload.png)
+{: .hands_on}
+
+## Step 2: Organize your data
+
+> <hands-on-title>Create a dataset collection (from empty history)</hands-on-title>
+> - Click on **Select item** at the top of the history panel
+> - Click on **Select all** (now visible at the top right)
+> - Click on **All ## selected**
+> - Choose **Auto build list**
+> - In the pop-up window:
+>   - Set the name: `4903881`
+>   - Turn off **Remove file extension**
+>   - Click on **Build**
 >
-> - 2c. Search “argo” in the top search bar. “Argo marine floats data and metadata from Global Data Assembly Centre (Argo GDAC)” appears in the label column just below
-> - 2d. Select it then “pub/dac/coriolis/4903881” then tick “4903881_meta.nc” + “profiles/”
-> - 2e. Click on “Select” on the bottom right of the pop-up. the list of the file to download is displayed
-> - 2f. Remove the Synthetic files (not useful for qualification & calibration actions)  by clicking on the corresponding garbage can on the right
-> - 2g. Click on “start” on the bottom of the pop-up. Once the download is complete (green color), files are stored in your history on the right.
+> Once the collection is created, all files turn green. You can limit visible files using the eye icons in the history panel.
 >
-> ![Image of the loading of the data files](../../images/bgc_calib/bgc_2f.png)
+> ![Create collection - empty history](../../images/bgc_calib/collection_emptyhist.png)
+> ![Collection build interface](../../images/bgc_calib/build_collection.png)
 {: .hands_on}
 
-> <hands-on-title>Organize your data</hands-on-title>
-> Create a dataset list/collection of the argo float 4903881 by clicking on
-> - 3a. “Select item” 
-> - 3b.  “Select all” (appears just after 3a and at the same level on the right)
-> - 3c.  “All ## selected”
-> - 3d.  “Auto build list” >> a pop-up is launched
-> - 3e.  Enter the name of the data collection : 4903881
-> - 3f.   turn off “Remove the file extension”
-> - 3g.  Click on “Build” on the bottom right of the pop-up
->
-> ![Image of how to build a collection from multiple netcdf files](../../images/bgc_calib/bgc_3a.png)
-> 
->
-> Once the collection is ready, everything is green in the history panel. It is possible to limit the history to useful files or collections by clicking on the eyes above the files list in the history.
+> <hands-on-title>Create a dataset collection (from non-empty history)</hands-on-title>

```

Comment:
Same here, is this an alternative to the previous create collection hands-on box? or do they need to do both?